pages/encrypt/index.jsx

```diff
@@ -21,14 +21,14 @@ module.exports = React.createClass({
           <meta property="og:type" content="website" />
           <meta property="og:title" content="Check out this awesome video"/>
           <meta property="og:site_name" content="Mozilla Advocacy - Encryption"/>
-          <meta property="og:url" content="https://advocacy.mozilla.org/encrypt/?video=2"/>
+          <meta property="og:url" content="https://advocacy.mozilla.org/encrypt/"/>
           <meta property="og:description" content="I just watched a great video about encryption. Watch it yourself and learn more about how encryption works in our everyday lives."/>
           <meta property="og:image" content="https://advocacy.mozilla.org/assets/Encryption-social-05-07.png" />
 
           <meta name="twitter:card" content="summary_large_image" />
           <meta name="twitter:site" content="@mozilla" />
-          <meta name="twitter:title" content="Meet Encryption" />
-          <meta name="twitter:description" content="I just learned how encryption works in our everyday lives. Check it out at mzl.la/encrypt via @mozilla #encrypt" />
```

Comment:
Where's the hashtag go? And I think we are moving from "Privacy lets you be you" for the title, though "Meet Encryption" was a late night guess for me. @pauljohnson?